.ruby-version

```diff
@@ -1 +1 @@
-2.2.2
\ No newline at end of file
+2.4.1
```

Comment:
Weren't we getting rid of these ruby version files in gems? I think Jacob is the one to ask.